app/views/govuk_publishing_components/components/layout_for_public/_header_logo.html.erb

```diff
@@ -0,0 +1,46 @@
+<%
+	hide_logo_text ||= false
+  large_navbar ||= false
+  logo_text ||= t("components.layout_super_navigation_header.logo_text")
+  logotype_classes = %w(govuk-header__logotype-crown gem-c-layout-super-navigation-header__logotype-crown)
+  logotype_classes << "gem-c-layout-super-navigation-header__logotype-crown--large-navbar" if large_navbar  
+%>
+
+<% unless hide_logo_text %>
+    <svg
+      focusable="false"
+      role="img"
+      class="govuk-header__logotype"
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 152 30"
+      height="30"
+      width="152"
```

Comment:
This missed the cut off for `govuk-frontend@5.0.0-beta.2` but is an [important VoiceOver fix](https://github.com/alphagov/govuk-frontend/pull/4497)

Thanks @patrickpatrickpatrick

```suggestion
      width="152"
      aria-label="GOV.UK"
```